core-web/libs/sdk/analytics/README.md

```diff
@@ -0,0 +1,115 @@
+# @dotcms/analytics
```

Comment:
Do we have a link to the Content Analytics docs? it will be nice to have it here.

Reply:
No yet.